### ISSUE TITLE
Is possible to set --installPath <dir>. 

### ISSUE THREAD

**Issue by JRSerjeant** [jberezanski/ChocolateyPackages#111]
Hello, 

I would like to define a custom install location for Visual Studio.

Using `Install-VisualStudio` from `chocolatey-visualstudio.extension` is it possible to set `--installPath <dir>`?

Thanks

**Comment by jberezanski** [comment]
Yes, it is. However, currently it can be done only by modifying the environment variable used by Chocolatey to pass package parameters:
```
$Env:chocolateyPackageParameters = '--installPath C:\VS'
Install-VisualStudio ...
```

**Comment by jberezanski** [comment]
I've wanted to implement a facility for passing such additional parameters directly to Install-VisualStudio (and Install-VisualStudioWorkload) for a long time, but never found the time to do it, unfortunately. I'd welcome a PR, though.

**Comment by jberezanski** [comment]
Available in extension 1.11.0 (currently in preview) as a new parameter of `Install-VisualStudio` and other public functions (`-DefaultParameterValues <hashtable>`).
